docs/csharp/language-reference/keywords/switch.md

```diff
@@ -46,7 +46,7 @@ Starting with C# 7.0, the match expression can be any non-null expression.
  
 ## The switch section
  
- A `switch` statement includes one or more switch sections. Each switch section contains one or more *case labels* followed by one or more statements. The following example shows a simple `switch` statement that has three switch sections. Each switch section has one case label, such as `case 1:`, and two statements.
+ A `switch` statement includes one or more switch sections. Each switch section contains one or more *switch labels* (either a case or default label) followed by one or more statements. The `switch` statement may include at most one default label placed in any switch section. The following example shows a simple `switch` statement that has three switch sections, each containing two statements. The second switch section contains the `case 2:` and `case 3:` labels.
```

Comment:
Let's revert the one change here from *case labels* to *switch labels*.

The only place where the term "switch label" appears is in the grammar. Everywhere else I looked in the C# spec, it's referred to as a "case label".